From 7547e12fc219c8a282ee812ad41ada721990003a Mon Sep 17 00:00:00 2001
From: Nobuhiko Tanibata <NOBUHIKO_TANIBATA@xddp.denso.co.jp>
Date: Wed, 10 Sep 2014 13:25:52 +0900
Subject: [PATCH 25/31] ivi-application.xml: Add configure event in ivi_surface

Configure event is notified to suggest resize of its surface

Signed-off-by: Nobuhiko Tanibata <NOBUHIKO_TANIBATA@xddp.denso.co.jp>
---
 protocol/ivi-application.xml | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/protocol/ivi-application.xml b/protocol/ivi-application.xml
index 80ab790..0c3fb8c 100644
--- a/protocol/ivi-application.xml
+++ b/protocol/ivi-application.xml
@@ -41,6 +41,24 @@
             </description>
             <arg name="visibility" type="int"/>
         </event>
+
+	<event name="configure">
+	    <description summary="suggest resize">
+	      The configure event asks the client to resize its surface.
+
+	      The size is a hint, in the sense that the client is free to
+	      ignore it if it doesn't resize, pick a smaller size (to
+	      satisfy aspect ratio or resize in steps of NxM pixels).
+
+	      The client is free to dismiss all but the last configure
+	      event it received.
+
+	      The width and height arguments specify the size of the window
+	      in surface local coordinates.
+	    </description>
+	    <arg name="width" type="int"/>
+	    <arg name="height" type="int"/>
+	</event>
     </interface>
 
     <interface name="ivi_application" version="1">
-- 
1.9.3

